app/daos/CrewDao.scala

```diff
@@ -72,6 +73,7 @@ class MongoCrewDao extends CrewDao {
       Json.obj("name" -> crew.name)
     )), crew).map(_ => crew)
 
+  def delete(crew: Crew):Future[Boolean] = ???
```

Comment:
sbt throw an error if there is no delete function in the `MongoCrewDao`.  The question is whether we should delete this Dao, because the code is not longer used.